### PR TITLE
Issue/jperf 548 fix flaky boards filtering

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/StaleElementMock.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package com.atlassian.performance.tools.jirasoftwareactions
+
+import org.openqa.selenium.*
+
+internal class StaleElementMock(private val webElement: WebElementMock) : WebElement by WebElementMock(emptyMap()) {
+    var isStale = false
+
+    override fun findElement(by: By?): WebElement {
+        return if (isStale) {
+            throw StaleElementReferenceException("stale element reference: element is not attached to the page document")
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/StaleElementMock.kt]: This looks like the production exception, nice touch

File: src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/BrowseBoardsPageTest.kt [hunk 2]
```diff
@@ -12,12 +13,23 @@ class BrowseBoardsPageTest {
...
             )
         )
 
         val boardIds = browseBoardsPage.getBoardIds()
 
         assertThat(boardIds).doesNotContainNull()
     }
+
+    @Test
+    fun shouldWorkWithAsynchronousFiltering() {
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/BrowseBoardsPageTest.kt]: That's a 43 ms repro. Well done

File: src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/StaleElementMock.kt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        } else {
+            webElement.findElement(by)
+        }
+    }
+
+    override fun isEnabled(): Boolean {
+        return if (isStale) {
+            throw StaleElementReferenceException("stale element reference: element is not attached to the page document")
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/StaleElementMock.kt]: We can deduplicate the exception

File: src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/StaleElementMock.kt (hunk 1 above)
Comment (wyrzyk) [reply src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/StaleElementMock.kt]: It mimics the real problem (see the stack trace below):

```2019-08-02T05:49:50,815Z ERROR {} [applying load #c341072c-30e2-464d-ad45-012b43ae3429] [ExploratoryVirtualUser] Failed to run com.atlassian.performance.tools.jirasoftwareactions.api.actions.BrowseBoardsAction@45427971, but we keep running
java.lang.Exception: Failed to run com.atlassian.performance.tools.jirasoftwareactions.api.actions.BrowseBoardsAction@45427971
	at com.atlassian.performance.tools.virtualusers.ExploratoryVirtualUser.runWithDiagnostics(ExploratoryVirtualUser.kt:82) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.ExploratoryVirtualUser.applyLoad(ExploratoryVirtualUser.kt:56) [jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.LoadTest.applyLoad(LoadTest.kt:164) [jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.LoadTest.access$applyLoad(LoadTest.kt:36) [jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.LoadTest$applyLoad$$inlined$forEach$lambda$1.run(LoadTest.kt:111) [jira-software-tests-tests.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_222]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_222]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
Caused by: org.openqa.selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: headless chrome=69.0.3497.0)
  (Driver info: chromedriver=2.43.600233 (523efee95e3d68b8719b3a1c83051aa63aa6b10d),platform=Linux 4.4.0-1067-aws x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'ip-10-0-0-177', ip: '10.0.0.177', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.0-1067-aws', java.version: '1.8.0_222'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.43.600233 (523efee95e3d68..., userDataDir: /tmp/.org.chromium.Chromium...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:35664}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: none, platform: LINUX, platformName: LINUX, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 69.0.3497.0, webStorageEnabled: true}
Session ID: cceaeaa679c2cc7aa65e9c57f36ff908
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_222]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_222]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_222]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_222]
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:279) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.RemoteWebElement.getAttribute(RemoteWebElement.java:128) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.jirasoftwareactions.api.page.BrowseBoardsPage.getBoardIds(BrowseBoardsPage.kt:31) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.jirasoftwareactions.api.page.BrowseBoardsPage.getScrumBoardIds(BrowseBoardsPage.kt:37) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.jirasoftwareactions.api.actions.BrowseBoardsAction.run(BrowseBoardsAction.kt:63) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.ExploratoryVirtualUser.runWithDiagnostics(ExploratoryVirtualUser.kt:79) ~[jira-software-tests-tests.jar:?]
	... 9 more
```

I'm running the regression test on the snapshot version, to confirm the reproduction is ok.

Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/StaleElementMock.kt]: FYI you can hide big content like this.
<details>
<summary>Big stacktrace</summary>

```
java.lang.Exception: Failed to run com.atlassian.performance.tools.jirasoftwareactions.api.actions.BrowseBoardsAction@45427971
	at com.atlassian.performance.tools.virtualusers.ExploratoryVirtualUser.runWithDiagnostics(ExploratoryVirtualUser.kt:82) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.ExploratoryVirtualUser.applyLoad(ExploratoryVirtualUser.kt:56) [jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.LoadTest.applyLoad(LoadTest.kt:164) [jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.LoadTest.access$applyLoad(LoadTest.kt:36) [jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.LoadTest$applyLoad$$inlined$forEach$lambda$1.run(LoadTest.kt:111) [jira-software-tests-tests.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_222]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_222]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
Caused by: org.openqa.selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: headless chrome=69.0.3497.0)
  (Driver info: chromedriver=2.43.600233 (523efee95e3d68b8719b3a1c83051aa63aa6b10d),platform=Linux 4.4.0-1067-aws x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'ip-10-0-0-177', ip: '10.0.0.177', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.0-1067-aws', java.version: '1.8.0_222'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.43.600233 (523efee95e3d68..., userDataDir: /tmp/.org.chromium.Chromium...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:35664}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: none, platform: LINUX, platformName: LINUX, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 69.0.3497.0, webStorageEnabled: true}
Session ID: cceaeaa679c2cc7aa65e9c57f36ff908
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_222]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_222]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_222]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_222]
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:279) ~[jira-software-tests-tests.jar:?]
	at org.openqa.selenium.remote.RemoteWebElement.getAttribute(RemoteWebElement.java:128) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.jirasoftwareactions.api.page.BrowseBoardsPage.getBoardIds(BrowseBoardsPage.kt:31) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.jirasoftwareactions.api.page.BrowseBoardsPage.getScrumBoardIds(BrowseBoardsPage.kt:37) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.jirasoftwareactions.api.actions.BrowseBoardsAction.run(BrowseBoardsAction.kt:63) ~[jira-software-tests-tests.jar:?]
	at com.atlassian.performance.tools.virtualusers.ExploratoryVirtualUser.runWithDiagnostics(ExploratoryVirtualUser.kt:79) ~[jira-software-tests-tests.jar:?]
	... 9 more
```
</details>

<details>
<summary>Big table</summary>

jira | jira nodes | database | overall error average [%] | overall error spread [%] | max action error max [%] | max action error label [%] | max action error spread [%] | apdex average (0.0-1.0) | apdex spread (0.0-1.0) | throughput average [HTTP requests / second] | throughput spread [HTTP requests / second] | worth exploring? | reason
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
c5.2xlarge | 1 | m4.4xlarge | 83.06151821695863 | 3.0833127724319525 | 100.0 | Project Summary | 0.0 | 0.08192022352827825 | 0.042345119953752765 | 77.15978039582151 | 5.2226078830557725 | YES | high availability
c5.2xlarge | 2 | m4.4xlarge | 80.39193606828778 | 3.7826329792025604 | 100.0 | View Board | 0.0 | 0.21188563720294668 | 0.026076749910216107 | 40.287576442881715 | 18.154414866060655 | YES | high availability
c5.2xlarge | 3 | m4.4xlarge | 74.00587205109714 | 4.929432368393947 | 100.0 | Create Issue | 5.128205128205138 | 0.2824328081556997 | 0.016672845227062083 | 27.748879619677602 | 4.027633192943863 | YES | high availability
c5.2xlarge | 4 | m4.4xlarge | 52.54544938397774 | 6.277522284676095 | 100.0 | Browse Boards | 0.0 | 0.26746653566237355 | 0.0018840308130200767 | 29.339875412335847 | 8.645846368388426 | YES | adding a node made improved Apdex enough
c5.2xlarge | 5 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 6 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 7 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 8 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 9 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 10 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 11 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 12 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 13 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 14 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 15 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.2xlarge | 16 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.4xlarge | 1 | m4.4xlarge | 79.03634835368206 | 1.0744834852080913 | 100.0 | View Board | 0.0 | 0.20216010520719663 | 0.030582836677019498 | 29.174383949836965 | 13.761206524878506 | YES | high availability
c5.4xlarge | 2 | m4.4xlarge | 16.330137779950395 | 5.575018557746274 | 47.94871794871795 | Search with JQL | 7.116321998211767 | 0.20377765108986184 | 0.0345949292193507 | 49.207512347168176 | 8.347038285402718 | YES | high availability
c5.4xlarge | 3 | m4.4xlarge | 0.8897737070710185 | 0.18065010194355313 | 4.21792618629174 | View Backlog | 0.259683472068164 | 0.4592388876299516 | 0.0022626856392278794 | 152.72128961175684 | 8.09450659621541 | YES | high availability
c5.4xlarge | 4 | m4.4xlarge | 0.6097840544485857 | 0.051985691314753946 | 5.5762081784386615 | View Backlog | 0.42557649719473556 | 0.5188135237669469 | 0.01162161945592688 | 212.3364027387512 | 5.495994597145824 | YES | adding a node made improved Apdex enough
c5.4xlarge | 5 | m4.4xlarge | 0.6465278276667461 | 0.16249115819915916 | 4.145077720207254 | View Backlog | 0.0026358597421385355 | 0.6156941586825646 | 0.0015817638712486426 | 254.66560364217662 | 1.1689308248997463 | YES | adding a node made improved Apdex enough
c5.4xlarge | 6 | m4.4xlarge | 0.37963935833320733 | 0.05603022477957964 | 4.399524375743162 | View Backlog | 0.9725580836083307 | 0.6730421877915251 | 0.014938609201669562 | 297.8138665306893 | 11.515848036145314 | YES | adding a node made improved Apdex enough
c5.4xlarge | 7 | m4.4xlarge | 0.3348038824369023 | 0.06310766598791889 | 3.751339764201501 | View Backlog | 0.5659017664768142 | 0.7232493871341977 | 6.016687653940256E-4 | 304.5920988568853 | 12.175431702811693 | YES | adding a node made improved Apdex enough
c5.4xlarge | 8 | m4.4xlarge | 0.35931037742635236 | 0.10460834533874397 | 4.037610619469026 | View Backlog | 0.8762129489199415 | 0.7296342353783243 | 0.011605787728229289 | 306.34315794605527 | 5.1763518677810225 | YES | adding a node made improved Apdex enough
c5.4xlarge | 9 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.4xlarge | 10 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.4xlarge | 11 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.4xlarge | 12 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.4xlarge | 13 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.4xlarge | 14 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.4xlarge | 15 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.4xlarge | 16 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c4.8xlarge | 1 | m4.4xlarge | 59.735798981024615 | 16.932299292158042 | 100.0 | Create Issue | 0.0 | 0.26293398397810297 | 0.21110473675843427 | 71.11209439528024 | 17.109144542772867 | YES | high availability
c4.8xlarge | 2 | m4.4xlarge | 21.284260094743967 | 8.892518771551032 | 80.0 | View Backlog | 34.02298850574713 | 0.3810898914091652 | 0.11213715707237992 | 66.88400844143644 | 72.11375716409704 | YES | high availability
c4.8xlarge | 3 | m4.4xlarge | 3.7596498796603246 | 0.30618500522228764 | 14.723203769140165 | Search with JQL | 0.10114676790100674 | 0.43954479990194273 | 0.008098608812894537 | 106.7841629495518 | 3.055272376573015 | YES | high availability
c4.8xlarge | 4 | m4.4xlarge | 0.6277049548249253 | 0.49240532131144776 | 4.592901878914405 | View Backlog | 1.157319056828515 | 0.49634027712821394 | 0.012242734042175407 | 193.48561102877434 | 15.213024621960983 | YES | adding a node made improved Apdex enough
c4.8xlarge | 5 | m4.4xlarge | 0.38714063661334613 | 0.06503315592183512 | 5.924339757316202 | View Backlog | 1.2611273220830421 | 0.5521682970862478 | 0.0035488824749770886 | 251.47659265120976 | 3.6044449077348304 | YES | adding a node made improved Apdex enough
c4.8xlarge | 6 | m4.4xlarge | 0.4819908174036441 | 0.07529907354456306 | 6.1547836684948205 | View Backlog | 0.15831100535549147 | 0.596237562187809 | 0.004885609670225444 | 294.8909667735153 | 4.184414139372166 | YES | adding a node made improved Apdex enough
c4.8xlarge | 7 | m4.4xlarge | 0.3965199898352711 | 0.07458741596160756 | 5.382293762575452 | View Backlog | 0.8223785982064173 | 0.6276628284707011 | 0.0049327810901712565 | 319.5672075114875 | 7.686313975392693 | YES | adding a node made improved Apdex enough
c4.8xlarge | 8 | m4.4xlarge | 0.40318632062507265 | 0.015018480735174589 | 5.140640155189137 | View Backlog | 0.4677429589274551 | 0.6446698200238493 | 0.0017785859691399342 | 327.91488131086317 | 2.7428473530952715 | YES | adding a node made improved Apdex enough
c4.8xlarge | 9 | m4.4xlarge | 0.6353778079256074 | 0.2578670639179321 | 4.681933842239186 | View Backlog | 0.10298647381813364 | 0.6557069755722396 | 0.002952600254452098 | 313.4386484850136 | 6.868049925300113 | YES | adding a node made improved Apdex enough
c4.8xlarge | 10 | m4.4xlarge | 0.9559839059845155 | 0.011730282277819581 | 6.6492829204693615 | View Backlog | 2.029222130195805 | 0.6531952096734317 | 0.010188671068066868 | 277.4524530809931 | 45.09671343039486 | YES | adding a node made improved Apdex enough
c4.8xlarge | 11 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c4.8xlarge | 12 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c4.8xlarge | 13 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c4.8xlarge | 14 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c4.8xlarge | 15 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c4.8xlarge | 16 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 1 | m4.4xlarge | 70.94670934940213 | 9.514170040485823 | 100.0 | View Backlog | 18.78531073446328 | 0.33555679945268524 | 0.02913829026339526 | 26.063110629158594 | 2.0270045481344248 | YES | high availability
c5.9xlarge | 2 | m4.4xlarge | 7.19369193163471 | 0.696831369999324 | 24.73423104181432 | Search with JQL | 3.2364905640867505 | 0.38661212017134783 | 0.02600515280440774 | 90.74171871473924 | 14.558343247768363 | YES | high availability
c5.9xlarge | 3 | m4.4xlarge | 0.5065387868862713 | 0.11954939005646731 | 5.859375 | View Backlog | 0.313600352112676 | 0.5272950459790591 | 0.01394155254970375 | 230.92260743641225 | 22.090370936017365 | YES | high availability
c5.9xlarge | 4 | m4.4xlarge | 0.40447965460881546 | 0.12099862125694288 | 4.722366372599896 | View Backlog | 0.856049203530433 | 0.6777569524488248 | 0.0076034019390698315 | 292.77507469951695 | 45.25428263047854 | YES | adding a node made improved Apdex enough
c5.9xlarge | 5 | m4.4xlarge | 0.3727541576728784 | 0.13649874071069162 | 3.595890410958904 | View Backlog | 0.4374321668475547 | 0.7182999088657485 | 0.014215724808965269 | 305.7711797875172 | 8.653787029584635 | YES | adding a node made improved Apdex enough
c5.9xlarge | 6 | m4.4xlarge | 0.24764964921117827 | 0.061136921693459884 | 3.541315345699832 | View Backlog | 1.3319518528013576 | 0.7415569094231524 | 0.003957290054739437 | 310.60739086596914 | 8.664753222974753 | YES | adding a node made improved Apdex enough
c5.9xlarge | 7 | m4.4xlarge | 0.2817022219193589 | 0.04495073203417482 | 3.2329988851727984 | View Backlog | 0.41926787785709774 | 0.7432119535555524 | 0.005412893823687148 | 306.1268251091145 | 1.3151707922284572 | YES | adding a node made improved Apdex enough
c5.9xlarge | 8 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 9 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 10 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 11 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 12 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 13 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 14 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 15 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.9xlarge | 16 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.18xlarge | 1 | m4.4xlarge | 75.88973304750672 | 10.462100825552383 | 100.0 | View Backlog | 7.5268817204301115 | 0.37141974945597833 | 0.11018643768746689 | 28.362811180267535 | 0.21128012512458483 | YES | high availability
c5.18xlarge | 2 | m4.4xlarge | 8.40353839083336 | 2.90226026790525 | 29.282700421940927 | Search with JQL | 6.330723020810982 | 0.45165451123409256 | 0.00945599519965945 | 78.45263002363177 | 17.64394811042132 | YES | high availability
c5.18xlarge | 3 | m4.4xlarge | 0.8219626612418867 | 0.03871080745312794 | 5.057251908396947 | View Backlog | 0.741592228988809 | 0.523570545763723 | 0.05102984948172373 | 194.079143435761 | 30.690323110274335 | YES | high availability
c5.18xlarge | 4 | m4.4xlarge | 0.41256965169654963 | 0.04236043842245363 | 5.145278450363197 | View Backlog | 0.8334060345333087 | 0.6227897147923906 | 0.003522934753678153 | 285.28860713240454 | 4.425688699132763 | YES | adding a node made improved Apdex enough
c5.18xlarge | 5 | m4.4xlarge | 0.3874119103401482 | 0.04849751858908202 | 4.7841913676547065 | View Backlog | 0.5816597220850861 | 0.6765696604282585 | 0.0059959864094739634 | 316.2675372694073 | 3.9295953264765444 | YES | adding a node made improved Apdex enough
c5.18xlarge | 6 | m4.4xlarge | 0.30679642591073064 | 0.010225673698387394 | 4.1281912004345465 | View Backlog | 0.37679142439871205 | 0.7072985034197394 | 0.0016667016400255807 | 306.3988589812563 | 6.504013647280658 | YES | adding a node made improved Apdex enough
c5.18xlarge | 7 | m4.4xlarge | 0.3114534076020754 | 0.11191746594600888 | 3.934426229508197 | View Backlog | 0.918305584682404 | 0.7177978750564622 | 0.0035611180782924023 | 310.79302896380636 | 8.208507159320789 | YES | adding a node made improved Apdex enough
c5.18xlarge | 8 | m4.4xlarge | 0.23383735589950033 | 0.06946664808571046 | 3.205495134516314 | View Backlog | 1.0823528839855285 | 0.7287832041407156 | 0.011884963289641282 | 307.54562761560845 | 11.568655858128409 | YES | adding a node made improved Apdex enough
c5.18xlarge | 9 | m4.4xlarge | 0.44790989215158905 | 0.028290178134078692 | 4.073199527744982 | View Backlog | 0.018373713924307644 | 0.7302614674423583 | 0.01062320457560939 | 291.89562236143286 | 9.24742775013425 | YES | adding a node made improved Apdex enough
c5.18xlarge | 10 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.18xlarge | 11 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.18xlarge | 12 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.18xlarge | 13 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.18xlarge | 14 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.18xlarge | 15 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
c5.18xlarge | 16 | m4.4xlarge | - | - | - | - | - | - | - | - | - | NO | adding more nodes did not improve Apdex enough
</details>

You can edit my comment to see the source

**Pull request merged**